releasenotes/notes/jmxfetch-0-36-1-983ecfebc05f6a5c.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    JMXFetch upgraded to `0.36.1 <https://github.com/DataDog/jmxfetch/releases/0.36.1>`_
```

Comment:
nit: a bit hacky, but can you update the existing release note on 0.36.0 instead? (so that the changelog ends up with one entry saying that we upgraded to 0.36.1 pointing to both jmxfetch changelog entries)

Reply:
I think removing the old release note might also work.

Reply:
done